tests/parm/cpld_control_lndp.nml.IN

```diff
@@ -0,0 +1,366 @@
+&atmos_model_nml
```

Comment:
I wonder if this new namelist template file is needed or we can use (updated) parm/cpld_control.nml.IN? What's the difference between this template and cpld_control.nml.IN? Are lndp variables the only difference?

Reply:
I generalised the cpld_control.nml.IN and deleted cpld_control_lndp.nml.IN.